src/Symfony/Component/FeatureToggle/README.md

```diff
@@ -0,0 +1,75 @@
+ToggleFeature Component
+=======================
+
+The ToggleFeature component provides TODO
+
+**This Component is experimental**.
+[Experimental features](https://symfony.com/doc/current/contributing/code/experimental.html)
+are not covered by Symfony's
+[Backward Compatibility Promise](https://symfony.com/doc/current/contributing/code/bc.html).
+
+Usage
+-----
+
+This example implements the following logic :
+* **Grants** when the `feat` parameter equals to `true` in the query string.
+* **Denies** when the `feat` parameter equals to `false` in the query string.
+* **Abstain** when the `feat` parameter is not found in the query string. So it
+ fallbacks to `false`.
+
+```php
+<?php
+
+use Symfony\Component\FeatureToggle\Feature;
+use Symfony\Component\FeatureToggle\FeatureChecker;
+use Symfony\Component\FeatureToggle\FeatureCollection;
+use Symfony\Component\FeatureToggle\Strategy\RequestQueryStrategy;
+
+$features = new FeatureCollection([
+    new Feature(
+        name: 'new_feature',
+        description: 'My new feature',
+        default: false,
+        strategy: new RequestQueryStrategy('feat'),
+    ),
+];
+$featureChecker = new FeatureChecker(
+    features: $features,
+    whenNotFound: false,
+);
+
+if ($featureChecker->isEnabled('new_feature')) {
+    // Use the new feature
+} else {
+    // Use the legacy code
+}
+```
+
+Available strategies
+--------------------
+
+**AffirmativeStrategy** : Takes a list of `StrategyInterface` and stops at the first `Grant`.
```

Comment:
this strategy is currently implementing a `OR`. But there is no implementation of a `AND`

Reply:
agreed. Didn't think of a valid use case but this can be easily implemented using a `unanimous` strategy (to keep the same names)

Reply:
Added UnanimousStrategy